docs/content/latest/migrate/migrate-from-postgresql/migrate-schema.md

```diff
@@ -163,14 +163,14 @@ CREATE TABLE contacts (
 );
 ```
 
-## Using `ysql_dump`
+## Use `ysql_dump`
 
-The PostgreSQL utility, pg_dump, can be used to dump the schema of a database. However, this schema would need  to be slightly modified as described below. 
+The PostgreSQL utility `pg_dump` can be used to dump the schema of a database.
```

Comment:
Think we can still call out that the descriptions to the schema migration are mentioned on this page. Instead of "as described below" we can say "as described in the sections above".

Reply:
Changed